pkg/api/vm/handler.go

```diff
@@ -397,6 +428,85 @@ func (h *vmActionHandler) abortMigration(namespace, name string) error {
 	return nil
 }
 
+func (h *vmActionHandler) getMigratableNodes(rw http.ResponseWriter, namespace, name string) error {
+	vmi, err := h.vmiCache.Get(namespace, name)
+	if err != nil {
+		return err
+	}
+	nodes, err := h.getMigratableNodesByVMI(vmi)
+	resp := struct {
+		Nodes []string `json:"nodes"`
+	}{
+		Nodes: nodes,
+	}
+
+	b, err := json.Marshal(resp)
+	if err != nil {
+		return err
+	}
+	_, err = rw.Write(b)
```

Comment:
should we return `Content-type` `application/json`?
![1679292018447](https://user-images.githubusercontent.com/15064560/226259323-2269fe06-33ae-4c5c-91e9-903fd9efbd75.png)